t/46-containters.t

```diff
@@ -0,0 +1,6 @@
+use Test::Most;
+use Test::Warnings ':report_warnings';
+
+is(system('docker', 'build', '-t', 'docker_worker', 'docker/worker'), 0);
+is(system('docker run -t registry.opensuse.org/devel/openqa/containers15.2/openqa_worker | grep "API key and secret are needed" | wc -l'), 0, '1');
```

Comment:
With this you are not running the container you built, do you?

Reply:
No, you're right, mistake

Reply:
It is also not nice that the check description is just `'1'` and that the previous check has no description at all.

I'm also wondering whether we want to run this test as part of the usual test suite because it requires a setup which is not covered by simply installing `openQA-devel` and is for most smaller contributions also not relevant.